packages/loader/container-loader/src/container.ts

```diff
@@ -1592,7 +1592,11 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         });
 
         deltaManager.on("throttled", (warning: IThrottlingWarning) => {
-            this.raiseContainerWarning(warning);
+            let containerWarning = warning as ContainerWarning;
```

Comment:
I'd think that inlining code like that assumes raiseContainerWarning() itself should be removed from Container class, but I do not see it happening in this PR.

Reply:
If we can remove the `raiseContainerWarning` implementation from `Container` class in this PR we should.  This probably also means removing from `ContainerContext`.

If some customer is depending on it (though I don't see anything at a quick glance) then we should add some aggressive warning in the `raiseContainerWarning` implementation that this is not to be called.

Reply:
`raiseContainerWarning` within runtime seems to assume that `raiseContainerWarning` is always present on the context. If we are following versioning compat N-1 rules, we probably need to wait 1 version before removing context/container references, otherwise N-1 runtime will not be able to call `raiseContainerWarning` on N context.  @ChumpChief let me know if Im wrong here. If so, maybe with this PR we can at least mark deprecation on context/container side if appropriate, and add some aggressive warning as you suggested.

Reply:
Ah yeah, you're right.  Deprecation would be good.